system/bootstrap.php

```diff
@@ -37,63 +31,45 @@
  * @package    CodeIgniter
  * @author     CodeIgniter Dev Team
  * @copyright  2019-2020 CodeIgniter Foundation
- * @license    https://opensource.org/licenses/MIT	MIT License
+ * @license    https://opensource.org/licenses/MIT - MIT License
  * @link       https://codeigniter.com
  * @since      Version 4.0.0
  * @filesource
  */
 
-/*
+use CodeIgniter\CodeIgniter;
+use CodeIgniter\Config\DotEnv;
+use CodeIgniter\Services;
+use Config\App;
+use Config\Autoload;
+use Config\Modules;
```

Comment:
I created PHPStan rule for it at https://github.com/codeigniter4/CodeIgniter4/pull/3786, no need to check manually ;)